ggshield/cmd/iac/scan/all.py

```diff
@@ -79,9 +81,12 @@ def iac_scan_all(
     if not paths:
         return IaCSkipScanResult()
 
+    root = get_project_root_dir(directory)
+    relative_paths = [str(x.resolve().relative_to(root)) for x in paths]
```

Comment:
Would be nice if `relative_paths` was a list of `Path` instead of a list of `str`. Can we remove the `str(...)` call?

Reply:
This would require changes in pygitguardian to accept Path lists (currently it expects string lists). Do you want me to open a PR?